tutorials/gui/gui_skinning.rst

```diff
@@ -67,14 +67,23 @@ Every option is associated to:
 
 An example usage:
 
-::
+.. tabs::
+ .. code-tab:: gdscript GDScript
 
     var t = Theme.new()
     t.set_color("font_color", "Label", Color(1.0, 1.0, 1.0))
 
     var l = Label.new()
     l.set_theme(t)
 
+ .. code-tab:: csharp
+
+    Theme t = new Theme();
```

Comment:
I'd suggest to use `var` like ```var t = new Theme();``` as it's a more common practice and it looks similar to the GDScript counterpart. The same goes for the following lines.